examples/cf-stackdriver-example/main.go

```diff
@@ -92,16 +95,17 @@ func EnvHandler(rw http.ResponseWriter, req *http.Request) {
 		environment[key] = val
 	}
 
-	rw.Write([]byte("Environment:\n"))
-	envJSON := HandleError(json.MarshalIndent(environment, "", "  ")).([]byte)
-	rw.Write(envJSON)
-
-	rw.Write([]byte("\nRequest:\n"))
-	reqJSON := HandleError(json.MarshalIndent(req.Header, "", "  ")).([]byte)
-	rw.Write(reqJSON)
+	var output []byte
+	output = append(output, []byte("Environment:\n")...)
+	envJSON := PanicOnError(json.MarshalIndent(environment, "", "  ")).([]byte)
```

Comment:
Is there any chance this contains credentials someone might want to steal? If so, I think we have the responsibility of disabling the environment echoing.

Reply:
I removed this GET request, because I agree: it doesn't seem like a good security practice